hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSet.java

```diff
@@ -64,7 +64,7 @@
   private static final String DUMMY_IP_ADDR = "0.0.0.0";
 
   private void initializeVolumeSet() throws Exception {
-    volumeSet = new VolumeSet(UUID.randomUUID().toString(), conf);
+    volumeSet = new VolumeSet(UUID.randomUUID().toString(), null, conf);
```

Comment:
any reason why we are migrating to this constructor with the additional argument?
same question applies to all the other places where this change has been made.

Reply:
This is change related to old commit. Will fix it.